Mathlib/Order/ConditionallyCompleteLattice/Finset.lean

```diff
@@ -97,18 +97,24 @@ theorem sup'_eq_csSup_image [ConditionallyCompleteLattice β] (s : Finset α) (H
     exact Finset.le_sup' _ ha
 #align finset.sup'_eq_cSup_image Finset.sup'_eq_csSup_image
 
-theorem inf'_eq_csInf_image [ConditionallyCompleteLattice β] (s : Finset α) (H) (f : α → β) :
-    s.inf' H f = sInf (f '' s) :=
-  @sup'_eq_csSup_image _ βᵒᵈ _ _ H _
+theorem inf'_eq_csInf_image (s : Finset ι) (hs) (f : ι → α) : s.inf' hs f = sInf (f '' s) :=
+  sup'_eq_csSup_image (α := αᵒᵈ) _ hs _
 #align finset.inf'_eq_cInf_image Finset.inf'_eq_csInf_image
 
-theorem sup'_id_eq_csSup [ConditionallyCompleteLattice α] (s : Finset α) (H) :
-    s.sup' H id = sSup s := by rw [sup'_eq_csSup_image s H, Set.image_id]
+theorem sup'_id_eq_csSup (s : Finset α) (hs) : s.sup' hs id = sSup s := by
+  rw [sup'_eq_csSup_image s hs, Set.image_id]
 #align finset.sup'_id_eq_cSup Finset.sup'_id_eq_csSup
 
-theorem inf'_id_eq_csInf [ConditionallyCompleteLattice α] (s : Finset α) (H) :
-    s.inf' H id = sInf s :=
-  @sup'_id_eq_csSup αᵒᵈ _ _ H
+theorem inf'_id_eq_csInf (s : Finset α) (hs) : s.inf' hs id = sInf s :=
+  sup'_id_eq_csSup (α := αᵒᵈ) _ hs
 #align finset.inf'_id_eq_cInf Finset.inf'_id_eq_csInf
 
+variable [Fintype ι] [Nonempty ι]
+
+lemma sup'_univ_eq_csupr (f : ι → α) : univ.sup' univ_nonempty f = ⨆ i, f i := by
+  simp [sup'_eq_csSup_image, iSup]
+
+lemma inf'_univ_eq_cinfi (f : ι → α) : univ.inf' univ_nonempty f = ⨅ i, f i := by
+  simp [inf'_eq_csInf_image, iInf]
```

Comment:
I'm really quite surprise we don't already have these.

Reply:
So am I!